### PR TITLE
*: improve compaction documentation

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/8018.
Address https://github.com/coreos/etcd/issues/7607.

https://github.com/coreos/etcd/issues/7607 needs more monitoring doc.

@mitake @yudai

### CODE REVIEW

File: Documentation/op-guide/maintenance.md [hunk 1]
```diff
@@ -6,7 +6,17 @@ An etcd cluster needs periodic maintenance to remain reliable. Depending on an e
 
 All etcd maintenance manages storage resources consumed by the etcd keyspace. Failure to adequately control the keyspace size is guarded by storage space quotas; if an etcd member runs low on space, a quota will trigger cluster-wide alarms which will put the system into a limited-operation maintenance mode. To avoid running out of space for writes to the keyspace, the etcd keyspace history must be compacted. Storage space itself may be reclaimed by defragmenting etcd members. Finally, periodic snapshot backups of etcd member state makes it possible to recover any unintended logical data loss or corruption caused by operational error.
 
-## History compaction
+## History compaction: Raft
```
Comment (xiang90) [inline Documentation/op-guide/maintenance.md]: raft log retention. do not confuse our users here. compaction should always mean compaction on db.

Comment (gyuho) [reply Documentation/op-guide/maintenance.md]: Good point.

**Comment by mitake** [comment]
@gyuho thanks for working on this! LGTM

**Comment by xiang90** [comment]
lgtm

**Pull request merged**